src/libcore/ptr/mod.rs

```diff
@@ -248,14 +246,16 @@ pub(crate) struct FatPtr<T> {
 ///
 /// // create a slice pointer when starting out with a pointer to the first element
 /// let x = [5, 6, 7];
-/// let ptr = x.as_ptr();
```

Comment:
This may not be necessary. `rustc` is clever enough to distinguish `std::ptr` from `ptr`.

Reply:
If you mean the refactor from `ptr` to `raw_pointer`, I simply reworded it so that a beginner will not get confused by the exemple

Reply:
It isn't necessary indeed